tests/ui/extern/explicit_extern_abis.rs

```diff
@@ -0,0 +1,8 @@
+//@ compile-flags: -Z unstable-options --edition future
+
+#![feature(explicit_extern_abis)]
+
+extern fn foo() {}
+//~^ ERROR extern declarations without an explicit ABI are disallowed
+
+fn main() {}
```

Comment:
This test will probably be made obsolete by the revised feature flag test, and that's OK.